frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -164,9 +171,23 @@ const Tooltip = () => {
         />
         <Head>
           <PinnedLabel>
-            <TypeIcon icon={isFolder ? faFolder : faMinus} />
+            <TypeIcon
+              icon={
+                // Parent is not element tooltipped or tooltip is a folder
+                isFolder || parent !== label
```

Comment:
Hmm, I still have a hard time understanding this, is there a case for something that's `!isFolder` but `parent !== label`?

Reply:
I just tried again to leave either out, but that creates the following issue:
- if the child is a Concept tree node, then the parent will also be rendered with `fa-minus` (if we leave `parent !== label`
- if we leave `is-folder` then the "big" folders without a parent folder will be rendered with `fa-minus`

My goal here was to mimic the icons from the concept tree.

Reply:
Yeah we should definitely mimic the icons from the concept tree. Maybe this gets clearer extracted to a function and using some if statements instead of nested ternary?